playground/nuxt.config.ts

```diff
@@ -1,5 +1,5 @@
 import { defineNuxtConfig } from 'nuxt3'
-import tailwindModule from '..'
+import tailwindModule from '../src/module'
```

Comment:
This is intentional. You should use `dev:setup` before. Will update readme.
```suggestion
import tailwindModule from '..'
```